.github/workflows/deb_release.yml

```diff
@@ -0,0 +1,73 @@
+name: Debian packages release
+
+on:
+  workflow_dispatch:
+    inputs:
+      config:
+        description: JSON of options
+        type: string
+        default: '{}'
+        required: true
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Save the config options
+        run: |
+          echo '${{ inputs.config }}' | jq > config.json
+      - name: Archive the release configuration
+        uses: actions/upload-artifact@v3
+        with:
+          name: config.json
+          path: config.json
+      - name: Install dependencies
+        run: |
+          sudo apt update -qq
+          sudo apt install -qq -y bumpversion devscripts python3-launchpadlib
+          git clone https://git.launchpad.net/~checkbox-dev/checkbox/+git/support
+      - name: Checkout checkbox monorepo
+        uses: actions/checkout@v3
+        with:
+          repository: canonical/checkbox
+          fetch-depth: 0
+          path: src/checkbox
+      - name: Check for new commits
+        run: |
+          support/release/git/release_deb_monorepo.py clone --config config.json
+      - name: Bump versions
+        run: |
+          git config --global user.email "sylvain.pineau@canonical.com"
+          git config --global user.name "Sylvain Pineau"
+          support/release/git/release_deb_monorepo.py bump --part major
+      - name: Create changelog
+        run: |
+          support/release/git/release_deb_monorepo.py changelog
+      - name: Archive changelog
+        uses: actions/upload-artifact@v3
+        with:
+          name: Changelog
+          path: changelog
+      - name: Push release tags
+        run: |
+          support/release/git/release_deb_monorepo.py push
+      - name: Update the PPA recipe and kick-off the builds
+        env:
+          LP_CREDS: credentials
```

Comment:
I don't quite follow this one. a) I don't understand how it works (I thought `env` is setting prerequisites for the `run` so I didn't think `credentials` would exist) and b) I would have thought '${{ secrets.LP_CREDS }}' belongs here instead:

```yaml
env:
  LP_CREDS: "${{ secrets.LP_CREDS }}"
```

Reply:
Also, if presently the script requires a file called `credentials`, it'd be better to change it to accept an env var instead so there is no temptation to write them out to disk.

Reply:
it's a file actually https://help.launchpad.net/API/launchpadlib#Authenticated_access

Reply:
The credentials do not in my understanding need to be passed in a file.

Reply:
I've update the legacy lp scripts to get credentials from the env var instead